README.md

```diff
@@ -173,32 +173,37 @@ Here is a guide for a quick setup of your `values-custom.yaml` file.
 
 ##### Host and port
 
-You need to replace `YOUR_MONGODB_HOST_URL` by your MongoDB server host.
-You also need to change the `port` value according to your MongoDB setup. 
+The preferred way to configure MongoDB connection information is to replace `YOUR_MONGODB_HOST_URL` by the
+full connection string URI. See [MongoDB official documentation](https://docs.mongodb.com/v4.0/reference/connection-string/). <br/>
+You must also set `port` to `0`. <br/>
+Username and/or password can be provided in the URI, special characters must be URL-encoded. By doing this, you must set `usePassword` to `false` in the below section "Authentication".
```

Comment:
Because of this, I think we need to store the mongo host as a cluster secret, just like the username and password.

Reply:
Yes, we could add the URL to mongo secrets